.github/workflows/continuous-integration.yml

```diff
@@ -36,6 +40,10 @@ jobs:
     - name: Validate composer.json and composer.lock
       run: composer validate
 
+      # Because PHPCS 3 requires PHP 5.4
+    - run: composer require --dev -W squizlabs/php_codesniffer:^2.0 symfony/yaml:^2.0 phpdocumentor/reflection-docblock:^2.0
```

Comment:
This seems an overkill to me: we don't need to run CS checks on all jobs, running it once should be more than enough.

We could run in in a separate job and not as part of the matrix, so we could get a dedicated check (and not having all of them fail when all tests are green and there's a space left somewhere).

Reply:
Thing is, syntax is different in different PHP versions, and PHPCS will check according to that syntax. So IMHO it makes sense to run PHPCS on as wide as possible range of PHP versions. But as usual, if you confirm, I will remove this.